src/Forms/GridFieldSiteTreeEditButton.php

```diff
@@ -28,7 +28,7 @@ public function getColumnContent($gridField, $record, $columnName)
         // which can make the form readonly if no edit permissions are available.
 
         $data = ArrayData::create([
-            'Link' => $record->CMSEditLink(),
+            'Link' => $record->getCMSEditLink(),
```

Comment:
Unrelated change - that method name changed in an earlier PR and we didn't catch this instance of it.